tests/acceptance/features/bootstrap/Provisioning.php

```diff
@@ -1610,31 +1610,31 @@ public function userHasTriedToResetPasswordOfUserUsingTheProvisioningApi(?string
 	}
 
 	/**
-	 * @Given /^the administrator has deleted user "([^"]*)" using the provisioning API$/
+	 * @Given /^the administrator has deleted user "([^"]*)" using the Graph API$/
 	 *
 	 * @param string|null $user
 	 *
 	 * @return void
 	 * @throws Exception
 	 */
-	public function theAdministratorHasDeletedUserUsingTheProvisioningApi(?string $user):void {
+	public function theAdministratorHasDeletedUserUsingTheGraphApi(?string $user):void {
 		$user = $this->getActualUsername($user);
-		$this->deleteTheUserUsingTheProvisioningApi($user);
+		$this->deleteTheUserUsingTheGraphApi($user);
 		WebDavHelper::removeSpaceIdReferenceForUser($user);
 		$this->userShouldNotExist($user);
 	}
 
```

Comment:
This is already present in the graph context why are we duplicating the same thing? Please don't duplicate